app/models/user.rb

```diff
@@ -321,7 +321,7 @@ def power_of_attorney
   end
 
   def flipper_id
-    email || account_uuid
+    email.downcase || account_uuid
```

Comment:
pretty sure that the Identity provider was always sending lower case anyway, adding that just in case it changes.